playbooks/group_vars/webportals.yml

```diff
@@ -204,6 +205,7 @@ webportal_common_config_defaults:
   skynet_db_port: "{{ default_mongo_port }}"
   skynet_db_replicaset: "{{ portal_cluster_id }}"
   skynet_db_user: "admin"
+  ncmec_debug: True
```

Comment:
Do we need to set this when the reporting is disabled anyway?

Reply:
The way I understand it is that we want to list the defaults here, in this file because it will autosave the value in LastPass and it's easier to find all config values. If reporting is disabled this variable simply won't be used and it won't show up in the env file.

But yeah other than that I also don't really know how to respond to that question. It might be redundant and personally, I also don't really see why default variables should be listed here rather than in the template itself.

Reply:
Talked it over with Fil. Probably the best approach is to leave this one out of the default vars indeed.
Updated it.